### DISCUSSION TITLE
Coverage by functions in console

### DISCUSSION THREAD

**Discussion by jose-lp** [linux-test-project/lcov#226] [Q&A]
Is there a way to get coverage by features in console?

something like this:

````gcov -f gdca_files````

but in lcov

**Comment by henry2cox** [comment]
you want a text report rather than HTML?
If so:  no.  Not implemented.
But you could try a text-based browser such as lynx.

If all you want is summary data - then you can get that from `lcov --summary ...`.

If you want text-based table summaries such as you see at the top of each of the genhtml-generated result pages:
No - not implemented.  But you can implement it yourself rather easily via the `genhtml --criteria-script ...` callback mechanism.

Hope that helped.
Henry
